app/components/works/show/attached_file_component.html.erb

```diff
@@ -1,10 +1,10 @@
 <tr>
-    <td><%= filename %>
+    <td class="file-level" style="--depth: <%=depth+1%>rem"><li><%= basename %>
```

Comment:
Can you say more about using an `<li>` outside the context of an ordered or unordered list? I know browsers don't choke on it but I'm not sure if it's valid HTML.

Reply:
I was trying to match Astrid's design, which shows a bullet point in front of files.  Perhaps there is a better way to do this other than an `<li>` tag, or perhaps we just need to wrap the whole thing in a `<ul>` tag, though not sure if this is legal since the inner `<li>` elements will be in table rows

Reply:
OK. I wonder if we could build on what we're doing for folders and find a FontAwesome icon that works for files? https://fontawesome.com/ Maybe @astridu and/or @amyehodge could suggest one?

Reply:
Closest free icon I found was a filled in circle, but at it's smallest, it still looks a bit big: 

![Screen Shot 2023-01-12 at 10 41 30 AM](https://user-images.githubusercontent.com/47137/212152070-48353d17-519f-427d-a2a9-c62a26e34a7f.png)

Reply:
@peetucket Give https://fontawesome.com/docs/web/style/size a shot.

Reply:
If so, you could try inserting an HTML character so we get the literal bullet without the list semantics: `&bull;`

Reply:
@peetucket OK, super. Thanks for rolling with the punches there.